Content.Server/PAI/PAISystem.cs

```diff
@@ -118,4 +132,48 @@ public void PAITurningOff(EntityUid uid)
                 _metaData.SetEntityName(uid, proto.Name);
         }
     }
+
+    private void OnCracked(EntityUid uid, PAIComponent comp, BreakageEventArgs args)
+    {
+        if (comp.Cracked)
+            return;
+        comp.Cracked = true;
+
+        // Update the pai description
+        if (TryComp(uid, out MetaDataComponent? metadata))
+        {
+            var proto = metadata.EntityPrototype;
+            if (proto != null)
+                _metaData.SetEntityDescription(uid, proto.Description + " " + Loc.GetString("pai-system-pai-description-cracked"));
+        }
+
+        // Add a glitch accent to the pai
+        AddComp<GlitchAccentComponent>(uid);
+    }
+
+    private void OnDropped(EntityUid uid, PAIComponent comp, LandEvent args)
+    {
+        // If the pai is dropped, it is instantly cracked.
+        _damageable.TryChangeDamage(uid, new DamageSpecifier(_prototypeManager.Index<DamageTypePrototype>("Blunt"), 150));
+    }
+
+    private void OnDamageChanged(EntityUid uid, PAIComponent comp, DamageChangedEvent args)
+    {
+        // If the pai is repaired, uncrack it.
+        if (comp.Cracked && TryComp<DamageableComponent>(uid, out var damageableComponent) && damageableComponent.TotalDamage < 100)
+        {
+            comp.Cracked = false;
```

Comment:
pretty sure we have a repairable system and component which feels like a better way of handling this. the hardcoded threshold is hurting me